certbot-apache/certbot_apache/tests/configurator_test.py

```diff
@@ -399,9 +390,12 @@ def test_deploy_cert_enable_new_vhost(self):
         self.assertTrue(ssl_vhost.enabled)
 
     def test_deploy_cert_newssl(self):
-        self.config = util.get_apache_configurator(
-            self.config_path, self.vhost_path, self.config_dir,
-            self.work_dir, version=(2, 4, 16))
+        with mock.patch("certbot.util.get_os_info") as mock_osi:
```

Comment:
This is in fact a problem with current design, where the OS gets fingerprinted every time `constants.os_constant("whatever")` is called to get a distro based variable. I really like the idea of moving all of it to configurator (overrides), and this would make these test hacks unnecessary.